### ISSUE TITLE
[references] Add a LR Finder in the training scripts

### ISSUE THREAD

**Issue by fg-mindee** [mindee/doctr#478]
For specific architectures, the learning rate that we generally use seems to yield various performances. We need a more consistent method to select an appropriate value for it.

Adding an option to select the learning rate would be quite valuable!
- [x] PyTorch support (#703)
- [ ] TensorFlow support (#747)
